src/main/java/seedu/task/ui/TaskCard.java

```diff
@@ -70,6 +91,15 @@ private void setCardDetails() {
         openTime.setText(task.getOpenTime().toPrettyString());
         closeTime.setText(task.getCloseTime().toPrettyString());
         
+        actualOpenTime.setText(task.getOpenTime().toString());
+        actualCloseTime.setText(task.getCloseTime().toString());
+        
+        firstOpenBracket.setText("(");
+        firstCloseBracket.setText(")");
+        
+        secondOpenBracket.setText("(");
+        secondCloseBracket.setText(")");
```

Comment:
Please use `String.format` instead of splitting it into multiple Labels.
Reference: [Java Documentation](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#format-java.lang.String-java.lang.Object...-), [Format specifiers](https://sharkysoft.com/archive/printf/docs/javadocs/lava/clib/stdio/doc-files/specification.htm)